tests/functional/u/undefined/undefined_variable_py38.py

```diff
@@ -155,3 +155,12 @@ def __init__(self, value):
 
 
 dummy = Dummy(value=val if (val := 'something') else 'anything')
+
+def expression_in_ternary_operator_inside_container():
+    """Named expression in ternary operator: inside container"""
+    return [val2 if (val2 := 'something') else 'anything']
+
```

Comment:
```suggestion

def expression_in_ternary_operator_inside_container_tuple():
    """Same case, using a tuple"""
    return [(val3, val3) if (val3 := 'something') else 'anything']
```
This test is not strictly necessary but I have to admit understanding why the next test case is failing took me way too long, so I hope this help with future reader understanding 😄